core/src/main/scala/com/lightbend/paradox/ParadoxProcessor.scala

```diff
@@ -78,8 +78,10 @@ class ParadoxProcessor(reader: Reader = new Reader, writer: Writer = new Writer)
     }
 
     if (expectedRoots != roots.size)
-      throw new IllegalStateException(s"Expected $expectedRoots ToC roots (based on paradoxExpectedNumberOfRoots) but found ${roots.size}: " +
-        roots.map(_.label.path).mkString("[", ", ", "]"))
+      throw new IllegalStateException(
+        s"Found [${roots.size}] top-level pages (pages that do no have a parent in the Table of Contents), but expected [$expectedRoots].\n" +
+        S"If this is intentional, update the `paradoxExpectedNumberOfRoots` sbt setting to reflect the new expected number of roots.\n" +
```

Comment:
Compilation is unhappy about the capital `S`.

Reply:
```suggestion
        s"If this is intentional, update the `paradoxExpectedNumberOfRoots` sbt setting to reflect the new expected number of roots.\n" +
```

Reply:
that's what I get for editing code in the github ui :(